CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- Fix MetaLogger on elixir 1.10.
```

Comment:
What was fixed?

Please add an additional "Changed" entry indicating that the minimal Elixir version is 1.10.